packages/mcs-lite-ui/src/Avatar/styled-components.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Wrapper: React.ComponentType<*> = styled.div`
+  img,
+  svg {
+    display: inline-block;
+    border-radius: 50%;
+    border: 1px solid ${props => props.theme.color.grayDark};
```

Comment:
NOTE: temporarily use `grayDark` 

(`grayBase` is currently darker than `grayDark`)

https://github.com/MCS-Lite/mcs-lite/blob/8280e339b47042c30ebe3c58184ee741377f31b1/packages/mcs-lite-theme/src/theme.js#L4-L5

Reply:
#604